sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
This is the only example I have on my Drupal site, from page manager.

http://cgit.drupalcode.org/page_manager/tree/page_manager.info.yml